crates/red_knot_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -383,6 +399,16 @@ impl<'db> UseDefMap<'db> {
     }
 }
 
+/// Mapping of `{key: value}` in which:
+/// - The key is an `(x, y)` tuple:
+///   - The element `x` identifies an eager nested scope
```

Comment:
I think the `Scoped` part of `ScopedEagerNestedScopeId` means that it is an ID for the eager nested scope which is particular to this outer scope; that seems worth being clear about here.

Reply:
No longer relevant